uui-components/src/pickers/DataPickerRow.tsx

```diff
@@ -41,7 +40,7 @@ export class DataPickerRow<TItem, TId> extends React.Component<DataPickerRowProp
                 ...this.props.rawProps,
             } }
             ref={ this.rowNode }
-            cx={ [clickHandler && this.props.isFocused && uuiMod.focus, this.props.cx] }
+            cx={ [(clickHandler || this.props.isFoldable) && this.props.isFocused && uuiMod.focus, this.props.cx] }
```

Comment:
Do we have a case when we don't have a `clickHandler`, but have `this.props.isFoldable`?